pwa-shell/src/ui/content-list/ContentList.js

```diff
@@ -60,21 +63,24 @@ class ContentList extends PureComponent {
 
         { categoryList }
         { searchResults }
-        <Container fluid className="content">        
-        <div className="mt-4">
-          { loadingMessage }
-        </div>
-        {
-          selectedCategoryCodes && selectedCategoryCodes.length
-            ? (
-              contentList && contentList.length ?
-              <div>
-                { contentListItems }
-              </div>
-              : <div>Nessun articolo trovato</div>
-            )
-            : 'Nessun argomento selezionato. Seleziona almeno un argomento dal menu in alto a sinistra.'
-        }
+        <Container fluid className="content">
+          {
+            selectedCategoryCodes && selectedCategoryCodes.length
+              ? (
+                contentList && contentList.length ?
+                <InfiniteScroll
+                  pageStart={0}
+                  loadMore={fetchNextContentList}
```

Comment:
for a better readability, what about renaming `fetchNextContentList` to `loadMoreListItems`?

Reply:
changed and moved to the component file